src/components/BorrowerProfile/LendCta.vue

```diff
@@ -452,9 +531,21 @@ export default {
 				paddingBottom: this.isSticky ? `${this.wrapperHeight}px` : '0',
 			};
 		},
+		transitionEnterClasses() {
+			if (this.isSticky) {
+				return 'tw-transform tw-translate-y-7 md:tw-translate-y-7 lg:tw--translate-y-7';
+			}
+			return 'tw-transform tw-translate-y-7 md:tw--translate-y-7 lg:tw--translate-y-7';
+		},
+		transitionEnterToClasses() {
+			// eslint-disable-next-line max-len
+			return "[ 'tw-transform','tw-translate-y-0', { 'md:tw-translate-y-0': !isSticky } 'md:tw-translate-y-0','lg:tw-translate-y-0' ]";
```

Comment:
This will need to compare against `this.isSticky` like you have in transitionEnterClasses above

Reply:
Better yet, I just noticed that the md:tw-translate-y-0 is the same regardless of isSticky, so I removed this computed function and moved this back up above to the enter-to-class="".